feat: add `stats/base/ndarray/dnanmskmidrange` (#10083)
stdlib-js/stdlib: lib/node_modules/@stdlib/stats/base/ndarray/dnanmskmidrange/docs/repl.txt

{{alias}}( arrays )

>     Calculates the midrange of a one-dimensional double-precision floating-point

Reviewer: ```suggestion
    Calculates the mid-range of a one-dimensional double-precision floating-point
```

You need to hyphenate `mid-range` here and everywhere else it is referred to in docs.

    ndarray according to a mask, ignoring `NaN` values.

    If provided an empty ndarray or a mask with all elements set to `1`, the
    function returns `NaN`.

    Parameters
    ----------
    arrays: ArrayLikeObject<ndarray>
        Array-like object containing an input ndarray and a mask ndarray.

    Returns
    -------
    out: number

>         Midrange.

Reviewer 2: ```suggestion
        Mid-range.
```

>         Midrange.

Reviewer: ```suggestion
        Mid-range.
```
